apps/web/views.py

```diff
@@ -98,6 +99,7 @@ def contact_us(request):
         if serializer.is_valid():
             serializer.save()
             response_data = {'message': 'We have received your request and will contact you shortly.'}
+            send_slack_notification(message="A *new contact message* is received")
```

Comment:
Add the contact message details inside the slack message. Otherwise, it's not useful at all.